cmd/create/cluster/cmd.go

```diff
@@ -3375,3 +3377,32 @@ func getExpectedResourceIDForAccRole(hostedCPPolicies bool, roleARN string, role
 
 	return strings.ToLower(fmt.Sprintf("%s-%s-Role", rolePrefix, accountRoles[roleType].Name)), rolePrefix, nil
 }
+
+func getInitialValidSubnets(awsClient aws.Client, reporter *reporter.Object) ([]*ec2.Subnet, error) {
+	initialValidSubnets := []*ec2.Subnet{}
+	allSubnets, err := awsClient.GetSubnetIDs()
+	if err != nil {
+		return initialValidSubnets, err
+	}
+	for _, subnet := range allSubnets {
+		hasRHManaged := false
+		for _, tag := range subnet.Tags {
+			switch *tag.Key {
+			case tags.RedHatManaged:
+				boolValue, err := strconv.ParseBool(*tag.Value)
+				if err != nil {
+					reporter.Debugf("An error occurred whilst parsing bool value of '%s' tag, value '%s'", *tag.Key, *tag.Value)
```

Comment:
Can we wrap these derefs in `aws.StringValue` just for a bit of added safety. I know the risk is small, but just to be safe.